smart_card_connector_app/src/window-main.js

```diff
@@ -58,7 +58,34 @@ GSC.ConnectorApp.Window.AppsDisplaying.initialize();
 GSC.ConnectorApp.Window.DevicesDisplaying.initialize();
 GSC.ConnectorApp.Window.LogsExporting.initialize();
 
-GSC.PopupWindow.Client.setExplicitWindowTitle();
 GSC.PopupWindow.Client.showWindow();
 
+/** @const */
+var DATA_I18N_STR = 'data-i18n';
```

Comment:
"_STR" doesn't give much information here, and it's not clear that this constant specifies attribute name.
Maybe "I18_DATA_ATTRIBUTE"?